app.py

```diff
@@ -155,19 +159,21 @@ def getData():
 
 	#USE CONDITIONAL ALONG WITH UI INFORMATION RECEIVED FROM THE CLIENT TO SWITCH
 	#BETWEEN HEAT MAP AND INTERPOLATION ANALYSIS
-
+        
+        if heatmap == "True":
```

Comment:
in Javascript, boolean values are lower case ('true' and 'false'). Try to check for these lower case values and see if it works.